### ISSUE TITLE
Query - Unit in the D_cba matrix

### ISSUE THREAD

**Issue by nizaanth** [IndEcol/pymrio#149]
Hello all,

I  was using the impacts from D_cba matrix, to be specific  'exio3.impacts.D_cba.loc["Water Consumption Blue - Total"]'. And  I used 
exio3.impacts.unit.loc["Water Consumption Blue - Total"] to get its corresponding unit. It shows the following:

unit    Mm3
Name: Water Consumption Blue - Total, dtype: object

My question is  whether to interpret it is as  cubic megametre or million cubic metre? Any clarity on this would be very helpful.

**Comment by konstantinstadler** [comment]
yeah, that is confusing. In water it actually means 1E6 m3. This comes from the water providers of the water extensions
